src/modules/reducers/augmintToken.js

```diff
@@ -121,6 +121,9 @@ export const connectAugmintToken = () => {
                 info: info
             });
         } catch (error) {
+            if (process.env.NODE_ENV !== "production") {
```

Comment:
how does this work on the client side where `process` is not available?

Reply:
https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables

Reply:
niiiice! so we can wrap other console messages as well: https://github.com/Augmint/augmint-web/pull/84/files#diff-7044f10098e6272508f6e4613b3ebc0aR81